### ISSUE TITLE
Addon causes a large number of LUA errors

### ISSUE THREAD

**Issue by triglav-modular** [Vysci/LFG-Bulletin-Board#221]
**Describe the bug**
Addon causes a large number of LUA errors

**Version:**
v3.23

**BugSack report**
````
2393x LFGBulletinBoard/RequestList.lua:482: attempt to perform arithmetic on field '?' (a nil value)
[string "@LFGBulletinBoard/RequestList.lua"]:482: in function 'UpdateList'
[string "@LFGBulletinBoard/GroupBulletinBoard.lua"]:775: in function 'Entry'
[string "@LFGBulletinBoard/LibGPIToolBox.lua"]:124: in function <LFGBulletinBoard/LibGPIToolBox.lua:122>

Locals:
AnchorTop = "GroupBulletinBoardFrame_ScrollChildFrame"
AnchorRight = "GroupBulletinBoardFrame_ScrollChildFrame"
yy = 218.000055
count = 9
doCompact = 1
cEntrys = 1
w = 508
itemHight = 12.000001
(for generator) = <function> defined =[C]:-1
(for state) = <table> {
 1 = <table> {
 }
 2 = <table> {
 }
 3 = <table> {
 }
 4 = <table> {
 }
 5 = <table> {
 }
 6 = <table> {
 }
 7 = <table> {
 }
 8 = <table> {
 }
 9 = <table> {
 }
 10 = <table> {
 }
 11 = <table> {
 }
 12 = <table> {
 }
 13 = <table> {
 }
 14 = <table> {
 }
 15 = <table> {
 }
 16 = <table> {
 }
 17 = <table> {
 }
 18 = <table> {
 }
 19 = <table> {
 }
 20 = <table> {
 }
}
(for control) = 20
i = 20
req = <table> {
 IsFriend = false
 last = 1714223782.101000
 class = "WARRIOR"
 IsGuildMember = false
 IsPastPlayer = false
 name = "Radahn"
 message = "LFG [[50] The Ancient Egg (4787)][[51] Saving Sharpbeak (2994)]"
 start = 1714223782.101000
 IsHeroic = false
 dungeon = "SOTA"
 IsRaid = true
}
hi = nil
(*temporary) = nil
(*temporary) = "SOTA"
(*temporary) = true
(*temporary) = false
(*temporary) = true
(*temporary) = "attempt to perform arithmetic on field '?' (a nil value)"
GBB = <table> {
 GetLfgList = <function> defined @LFGBulletinBoard/LfgToolList.lua:172
 CreateTagList = <function> defined @LFGBulletinBoard/GroupBulletinBoard.lua:294
 GetDungeonNames = <function> defined @LFGBulletinBoard/Dungeons.lua:23
 ClearNeeded = false
 InitGroupList = <function> defined @LFGBulletinBoard/GroupList.Lua:175
 TBCMAXDUNGEON = 56
 LfgRequestHideTooltip = <function> defined @LFGBulletinBoard/LfgToolList.lua:765
 tagList = <table> {
 }
 suffixTags = <table> {
 }
 LevelRange = <function> defined @LFGBulletinBoard/GroupBulletinBoard.lua:117
 ParseMessage = <function> defined @LFGBulletinBoard/RequestList.lua:709
 LFG_Successfulljoined = true
 ClearTimer = 1714224010.203000
 UpdateLfgToolNoSearch = <function> defined @LFGBulletinBoard/LfgToolList.lua:268
 VanillaDungeonLevels = <table> {
 }
 NotifySound = 1210
 PostTbcDungeonLevels = <table> {
 }
 MiniIcon = "Interface\Icons\spell_holy_prayerofshadowprotection"
 searchTagsLoc = <table> {
 }
 FoldAllDungeon = <function> defined @LFGBulletinBoard/RequestList.lua:866
 ClickFrame = <function> defined @LFGBulletinBoard/RequestList.lua:910
 CreateChatFrame = <function> defined @LFGBulletinBoard/Chat.lua:3
 Initalized = true
 BtnClose = <function> defined @LFGBulletinBoard/GroupBulletinBoard.lua:238
 Options = <table> {
 }
 OnLoad = <function> defined @LFGBulletinBoard/GroupBulletinBoard.lua:746
 WOTLKDUNGEONSTART = 57
 dungeonSecondTags = <table> {
 }
 SplitNoNb = <function> defined @LFGBulletinBoard/GroupBulletinBoard.lua:60
 TAGBAD = "---"
 VanillDungeonNames = <table> {
 }
 GetDungeonSort = <function> defined @LFGBulletinBoard/Dungeons.lua:778
 TBCDUNGEONSTART = 32
 heroicTagsLoc = <table> {
 }
 LocalizationInit = <function> defined @LFGBulletinBoard/Localization.lua:741
 Popup_Minimap = <function> defined @LFGBulletinBoard/GroupBulletinBoard.lua:372
 ClickDungeon = <function> defined @LFGBulletinBoard/RequestList.lua:917
 Tool = <table> {
 }
 HeroicKeywords = <table> {
 }
 Raids = <table> {
 }
 RequestShowTooltip = <function> defined @LFGBulletinBoard/RequestList.lua:966
 OptionsInit = <function> defined @LFGBulletinBoard/Options.lua:129
 AllowInInstance = <function> defined @LFGBulletinBoard/GroupBulletinBoard.lua:47
 MSGPREFIX = "GBB: "
 UpdateLfgTool = <function> defined @LFGBulletinBoard/LfgToolList.lua:246
 OptionsUpdate = <function> defined @LFGBulletinBoard/Options.lua:77
 Announce = <function> defined @LFGBulletinBoard/Chat.lua:140
 LFG_UPDATETIME = 10
 PvpNames = <table> {
 }
 BtnSelectChannel = <function> defined @Interface
```

**Comment by juemrami** [comment]
Same issue as #214 

Heres a quick for you, run copy paste these individually in your chat:

`/run GroupBulletinBoard_Addon.DBChar["FilterDungeonEOTS"] = false;`
`/run GroupBulletinBoard_Addon.DBChar["FilterDungeonARENA"] = false;`
`/run GroupBulletinBoard_Addon.DBChar["FilterDungeonWG"] = false;`
`/run GroupBulletinBoard_Addon.DBChar["FilterDungeonSOTA"] = false;`

and `/reload`

You shouldn't get any more errors until a proper fix is implemented.

**Comment by juemrami** [comment]
Fixed in #219
